### ISSUE TITLE
iOS: Add swipe left on chats

### ISSUE THREAD

**Issue by TheDiamondYT1** [gitterHQ/gitter#1411]
Obviously this would only work for the iOS app.

---

By adding a way to swipe left on the chats you could quickly leave a chat, mark as read, and maybe another action. Here's an example from the stock mail app:

![image](http://i.imgur.com/HHhT4Uy.png "Swipe")

As you can see, this would be using the Gitter iOS app much quicker.

**Comment by TheDiamondYT1** [comment]
This is an android feature

**Comment by MadLittleMods** [comment]
@TheDiamondYT1 Fair point. Investigating some apps and Gmail kinda has something similar on Android.

**Comment by TheDiamondYT1** [comment]
Isint that swipe to delete though?

**Comment by TheDiamondYT1** [comment]
Let's look at Kik messenger:

**Android**
You hold on a chat for a few seconds and it displays a dialog with actions

**iOS**
You swipe left on the chat and it displays 3 options.

**Comment by TheDiamondYT1** [comment]
Bump
